mgmtsystem_manual/models/mgmtsystem_manual.py

```diff
@@ -0,0 +1,9 @@
+
+from odoo import models, fields
+
+
+class MgmtSystemManual(models.Model):
+
+    _inherit = 'mgmtsystem.system'
+    _description = 'Manual'
+    manual = fields.Many2one('document.page', string='Manual')
```

Comment:
You should add a domain, because all document pages can be selected

Reply:
It should be renamed as manual_id. And just a question. Why are you making it as part of a category? It can be changed...
Maybe, you should add a boolean like: is_mgmsystem_manual. It's default is false, but you add on the context that it is true on this view.